docs/application-connector/docs/010-details-ac-deployment.md

```diff
@@ -3,59 +3,131 @@ title: Deploy a new Remote Environment
 type: Details
 ---
 
-By default, Kyma comes with two Remote Environments preinstalled. Those Remote Environments are installed in the `kyma-integration` Namespace.
+The Remote Environment controller is responsible for managing all Remote Environment. 
```

Comment:
As above, it is rather responsible for managing RE infrastructure and not REs themselves